domain/src/main/java/org/oppia/android/domain/topic/TopicController.kt

```diff
@@ -485,6 +485,7 @@ class TopicController @Inject constructor(
               availableToPlayNow = true
             } else availableToPlayInFuture = true
           }.build()
+          enablePracticeTab = topicRecord.enablePracticeTab
```

Comment:
```suggestion
          hasPracticeQuestions = topicRecord.enablePracticeTab
```

The controller should not need context on what we intend to do with this variable, the viewmodel will make the decision. here, we only need to know whether the topic has practice questions.

Reply:
Done.